packages/vite-plugin-cloudflare/package.json

```diff
@@ -9,23 +17,32 @@
 	"license": "MIT",
 	"sideEffects": false,
 	"type": "module",
-	"main": "dist/index.js",
-	"types": "dist/index.d.ts",
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"import": "./dist/index.js"
+		}
+	},
```

Comment:
out of curiosity, is `exports` needed when we have only the index files? (and they are already specified by `main` and `types`)

Reply:
Not strictly. If we were to choose though I would go with `exports` as it's the more modern/encouraged approach and makes it easier to add additional exports in future. I left `main` in because there's no harm in doing so (`exports` is prioritised) but I can remove it if you like.

Reply:
ah no, no problem I was just curious 🙂 

(and I can't never remember which is the new/modern one and which is the old one 😓)